src/components/Article/ArticleStyles.tsx

```diff
@@ -1,26 +1,36 @@
 import { createStyles, Theme } from '@material-ui/core/styles';
 export const styles = (theme: Theme) =>
   createStyles({
+    fixed: {
+      position: 'fixed',
+      top: '60px',
+      width: '100vw',
```

Comment:
probably won't need to define width to be 100vw since it should be handled in higher level components (ie WCSRoute)!